stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkPeerManager.java

```diff
@@ -21,23 +21,31 @@
 
   private final ResponseBodyFileManager mResponseBodyFileManager;
 
+  private final AsyncPrettyPrinterRegistry mAsyncPrettyPrinterRegistry;
```

Comment:
@rickbrew the peer manager is the only context given to `NetworkEventReporterImpl`.  `Network` owns/constructs the peer manager.  `NetworkEventReporterImpl` accesses the instance that was constructed.

They talk to each other through a singleton because I didn'tw ant to ask third party callers to have to keep track of the default modules provider that they passed in through `Stetho.initialize`.